charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v0.16.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 6.7.1
+version: 6.7.2
```

Better bump to 6.8.0 since a new feature introduced.

```suggestion
version: 6.8.0
```

---

Thank you for the review.
The version update to `6.8.0` seems indeed more appropriate.